test/integration/dataunion/withdraw.test.ts

```diff
@@ -131,13 +139,39 @@ const testWithdraw = async (
     const stats = await memberClient.getDataUnion(dataUnion.getAddress()).getMemberStats(memberWallet.address)
     log(`Stats: ${JSON.stringify(stats)}`)
 
-    // test setup done, do the withdraw
-    const balanceBefore = await getBalance(memberWallet)
+    const getRecipientBalance = async () => (
+        options.sendToMainnet
+            ? memberClient.getTokenBalance(recipient)
+            : memberClient.getSidechainTokenBalance(recipient)
+    )
+
+    const balanceBefore = await getRecipientBalance()
     log(`Balance before: ${balanceBefore}. Withdrawing tokens...`)
 
-    const withdrawTr = await withdraw(dataUnion.getAddress(), memberClient, memberWallet, adminClient)
-    log(`Tokens withdrawn, sidechain tx receipt: ${JSON.stringify(withdrawTr)}`)
-    const balanceAfter = await getBalance(memberWallet)
+    // "bridge-sponsored mainnet withdraw" case
+    if (!options.payForTransport && options.waitUntilTransportIsComplete) {
+        log(`Adding ${recipient} to bridge-sponsored withdraw whitelist`)
+        bridgeWhitelist.push(recipient)
```

Comment:
Does `recipient` need to be `toLowerCase`?

Reply:
it needs to eventually be, it's toLowerCased in the "simulated bridge". I think it's good to keep the checksum-case where possible.